src/api/institutionApi.ts

```diff
@@ -0,0 +1,81 @@
+import Axios from 'axios';
+import { API_URL } from '../utils/constants';
+import {
+  Institution,
+  InstitutionSubUnit,
+  emptyInstitution,
+  emptyInstitutionSubUnit,
+  InstitutionPresentationModel,
+  emptyInstitutionName,
+} from '../types/institution.types';
+
+export enum InstituionApiPaths {
+  QUERY = '/institution',
+  GET = '/institution/unit/',
+}
+
+export const queryInstitution = async (searchTerm: string) => {
+  try {
+    const response = await Axios({
+      method: 'GET',
+      url: `${API_URL}${InstituionApiPaths.QUERY}?name=${searchTerm}`,
+    });
+    return response.data;
+  } catch (e) {
+    console.log(e);
+    return [];
+  }
+};
+
+export const getInstitutionSubUnit = async (cristinUnitId: string) => {
+  try {
+    const response = await Axios({
+      method: 'GET',
+      url: `${API_URL}${InstituionApiPaths.GET}${cristinUnitId}`,
+    });
+    return response.data;
+  } catch {
+    return [];
+  }
+};
+
+export const institutionLookup = async (cristinUnitId: string) => {
+  const idElements = cristinUnitId.split('.');
```

Comment:
skjønner ikke helt navnet på konstanten her. hva med institutionNumber ellernoe?

Reply:
hva med å lage en funksjon som tar inn cristinUnitId og returnerer institituinID ? Da får du forklart hva som skjer og kan gjemme unna ekkel syntax :-)

Reply:
Dette er kun en midlertidig løsning til backend har en lookup for institusjoner.